index.js

```diff
@@ -301,6 +301,7 @@ function focusTrap(element, userOptions) {
   }
 
   function tryFocus(node) {
+    if (!state.active) return;
```

Comment:
The fix you're proposing here seems right, but I don't like that we're checking for the activate state in `tryFocus()`. I think the fix belongs more in the timeout related to the delayed focus on activation, [at line 145](https://github.com/focus-trap/focus-trap/blob/master/index.js#L145), just before calling `tryFocus()`. 

`tryFocus()` itself doesn't need to be concerned about state.

And with newer options in `focus-trap`, I thought the new `delayInitialFocus` option set to `false` might help since your description makes it sound like that's basically the problem (because you say the trap is deactivated by the time the initial focus is set), but that didn't help either. Maybe I misinterpreted something there.

Reply:
@cgood92 Since you're around... any thoughts on this one?

Reply:
Cool, thanks!